docs/androidx_test.md

```diff
@@ -236,6 +236,7 @@ UI threads.
 tests. Since Robolectric 4.0, Espresso APIs are now supported in Robolectric tests.
 
 /// tab | Java
+
```

Comment:
Is it on purpose to have all these chances?

Reply:
Generated by markdown lint cli. Looks like it is reasonable?

Reply:
Maybe we should move them in a dedicated PR?

Reply:
I removed related changes. If we use markdown-cli to format markdown files, maybe we need to make it on CI and validate every changes.

Reply:
It's already the case: https://github.com/robolectric/robolectric.github.io/actions/runs/16264879646/job/45918198519?pr=467

But apparently it doesn't detect Markdown files anymore 🤔

Reply:
I use the command in `README.md`, maybe it is related to cli version.

Reply:
I've submitted a PR to fix that: #471